src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
+      <div class="calendar__day calendar__day--d-1"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
try to implement this task without those classes